### PR TITLE
feat: add basic html structure

### LINKED ISSUE #34: HTML | Create Basic Structure
Prepare basic structure for project

### PR DESCRIPTION
**Pull request by vieraboschkova**
closes #34

### CODE REVIEW

**Review by paula0403 — COMMENTED** [review]
Class names should not be added to semantic tags.
Class name "header" in in tag header isn't good practise. 
All elements should be in div with id="swquiz-app" in all files. 
You can use dash instead underscore in class names, it will be better.
If we need elements to JS logic, maybe will be better use ID's instead classes.

**Comment by vieraboschkova** [comment]
> Class names should not be added to semantic tags.
Can you give a source for this? I based it on: https://dev.to/kenbellows/stop-using-so-many-divs-an-intro-to-semantic-html-3i9i
> Class name "header" in in tag header isn't good practise.
Please propose another.
> All elements should be in div with id="swquiz-app" in all files.
> You can use dash instead underscore in class names, it will be better.
SCSS is based on elements(__) and modifiers(--). As color is just a change, modification, I used '--' for red.
> If we need elements to JS logic, maybe will be better use ID's instead classes.
After structure is discussed and accepted as our path for further development, the id's should be added.

**Review by DominikNowak — CHANGES_REQUESTED** [review]
I think it's ok in general. There is header with class 'header' in every file, I don't know if this will not cause any collision. How about naming these classes 'game-header', 'settings-header' etc. for safety? That would also meet Paulina's requirements. What do you think also about create folder 'pages' and move all these files in there?

File: scoreBoards copy.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
```
[the remark this answers is not in the document]
Comment (vieraboschkova) [reply scoreBoards copy.html:1]: > What do you think also about create folder 'pages' and move all these files in there?
---> Definitely a good idea.

**Comment by vieraboschkova** [comment]
Changes in the project's architecture

**Pull request closed without merging**